packages/storybook/src/mocks/MockParamsProvider.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+import { useLocation, ParamsContext, parseSearch } from '@redwoodjs/router'
+
+interface Props {
+  path?: string
```

Comment:
Did you intend for this to be `allParams` to match `ParamsProvider` in `@redwoodjs/router`?

Reply:
In general I don't like to include "cleanup" stuff in a PR. But I *love* cleanup work! :D So what I do is I create a separate PR for the cleanup, to keep my main PR focused on just a single thing.

https://github.com/redwoodjs/redwood/pull/10699

Reply:
@arimendelow can you please update the code here to match the new code in packages/testing/src/web/MockParamsProvider.tsx?

Reply:
makes sense :) done!!